src/components/pages/HomePage.tsx

```diff
@@ -0,0 +1,5 @@
+export const HomePage = () => {
+  return (
```

Comment:
when you have only jsx , return without word `return` , in round brackets `HomePage = () => (`
(if linter allows)